project/make/.dockerversion

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+rm -rf autogenerated
+mkdir -p autogenerated/dockerversion
+cat > autogenerated/dockerversion/dockerversion.go<<EOF
+	// AUTOGENERATED FILE; see hack/make.sh
+	package dockerversion
+	var (
+		GITCOMMIT string = "$GITCOMMIT"
+		VERSION string = "$VERSION"
+	)
```

Comment:
This indentation will likely compile correctly, but IMO if we're generating Go code, it should be reasonably `gofmt`ed (ie, `gofmt -s -w` on the result should be unchanged).